spec/features/bidder_interacts_with_auction_spec.rb

```diff
@@ -20,6 +20,7 @@
       visit auction_path(@published_auction)
 
       expect(page).to have_text(@published_auction.description)
+      expect(page).to have_text(@published_auction.title)
```

Comment:
If you also wanted to make sure the title was inside an `h1` tag, you could do this:
```
have_css('h1', text: @published_auction.title) 
```